geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/TiledRasterLayer.scala

```diff
@@ -314,6 +315,26 @@ abstract class TiledRasterLayer[K: SpatialComponent: JsonFormat: ClassTag: Bound
       case multi: MultiPolygon => rdd.polygonalSumDouble(multi)
     }
 
+  def tobler(
+    zFactor: Double,
+    targetCell: String
+  ): TiledRasterLayer[K] = {
+    val n = Square(1)
+    val target = TileLayer.getTarget(targetCell)
+    val singlebandRDD: TileLayerRDD[K] = rdd.withContext(_.mapValues(_.band(0)))
+    val cellSize = singlebandRDD.metadata.layout.cellSize
+
+    val toblerCalc = (tile: Tile, bounds: Option[GridBounds], cellSize: CellSize) =>
+      Tobler(tile, n, bounds, cellSize, zFactor, target)
+
+    val calc = (tile: Tile, bounds: Option[GridBounds]) => toblerCalc(tile, bounds, cellSize)
```

Comment:
The indirection here seems strange. Why not call `Tobler` directly?

Reply:
I didn't call `Tobler` directly because because I was using `FocalOperation` instead. If that's not clear, though, I could use it directly.